### PR TITLE
fixes rotating chairs not updating their layer

### PR DESCRIPTION
**Pull request by PKPenguin321**
ugh

FIxes https://github.com/tgstation/tgstation/issues/23112

### CODE REVIEW

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 1]
```diff
@@ -55,17 +55,21 @@
...
 			buckled_mob.setDir(direction)
 
-/obj/structure/chair/post_buckle_mob(mob/living/M)
+/obj/structure/chair/proc/handle_layer()
 	if(has_buckled_mobs() && dir == NORTH)
 		layer = ABOVE_ALL_MOB_LAYER
 	else
 		layer = OBJ_LAYER
 
+/obj/structure/chair/post_buckle_mob(mob/living/M)
```
Comment (ChangelingRain) [inline code/game/objects/structures/beds_chairs/chair.dm]: this should probably call ..() just as a standard

Comment (PKPenguin321) [inline code/game/objects/structures/beds_chairs/chair.dm]: good point

Comment (PKPenguin321) [inline code/game/objects/structures/beds_chairs/chair.dm]: did it but the comment didnt close

**Comment by MrPerson** [comment]
This kills tactical chairs

**Comment by PKPenguin321** [comment]
I did that joke two PRs ago this is the fallout from their death

**Comment by MrPerson** [comment]
It's a permajoke. It will never get old, ever.

**Pull request merged**